src/main/java/org/orecruncher/dsurround/config/dimension/DimensionInfo.java

```diff
@@ -28,7 +27,7 @@
 
     public DimensionInfo(final World world, final DimensionConfig dimConfig) {
         // Attributes that come from the world object itself. Set now because the config may override.
-        DimensionType dt = world.getDimension();
+        world.getDimension();
```

Comment:
I'm pretty sure calling a getter and discarding its return value is useless, why not remove the entire line?